helix-vcs/src/git.rs

```diff
@@ -17,14 +18,17 @@ impl Git {
         // custom open options
         let mut git_open_opts_map = git::sec::trust::Mapping::<git::open::Options>::default();
 
-        // don't use the global git configs (not needed)
+        // On windows various configuration options are bundled as part of the installations
+        // This path depends on the install location of git and therefore requires some overhead to lookup
+        // This is basically only used on windows and has some overhead hence it's disabled on other platforms.
+        // `gitoxide` doesn't use this as default
```

Comment:
`gix` is a good example for it as it selectively enables this option depending on which command is run. Some are known to need this extra information, like when it's related to authentication on all platforms, whereas others don't.